examples/workshop-r-shiny-styling/.saturn/saturn.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "workshop-r-shiny-styling",
+  "image_uri": "public.ecr.aws/saturncloud/saturn-rstudio:2022.04.01",
+  "description": "Supporting materials for the Styling Shiny workshop",
+  "working_directory": "/home/jovyan/examples/examples/workshop-r-shiny-styling",
+  "extra_packages": {
+    "cran": {
+      "install": "shiny"
+    }
+  },
+  "git_repositories": [
+    {
+      "url": "git@github.com:saturncloud/examples.git",
+      "path": "/home/jovyan/examples"
+    }
+  ],
+  "rstudio_server": {
+    "instance_type": "large",
+    "disk_space": "10Gi",
+    "auto_shutoff": "1 hour"
+  },
+  "version": "2022.04.01"
```

Comment:
All examples in the examples repo use the same version. This is mostly to keep things internally consistent so that we don't get confused. That is enforced by CI. So this needs to be "2022.01.06"